doc/nrf/gs_installing.rst

```diff
@@ -55,6 +55,61 @@ The installation process is different depending on your operating system.
 
       Also see :ref:`zephyr:mac-setup-alts` for additional information.
 
+If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate the Ninja files that the |NCS| uses.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      The GN, as well as the Connected Home over IP project, do not support Windows yet.
```

Comment:
Right, that was simplified too much. I rephrased it a little. Is it ok now?

Reply:
This I find confusing now.
If GN is actually supported on Windows and the limitation is in CHIP, we should tell users how to install it here and move the information that CHIP is not supported on Windows to the CHIP user guide. Maybe with a note here to prevent users from planning to use CHIP on Windows.

Reply:
But GN is currently only required by CHIP, so if CHIP is not supported on Windows, we shouldn't tell a user to install GN needlessly.

Reply:
I thought the plan was to just have everybody install it?

Reply:
I'm not aware of such plans. :) I thought that's why there's the following condition at the beginning of the section: "If you are interested in building `Connected Home over IP`_ applications". I believe there are currently no other use cases for GN than building CHIP applications.

Reply:
> I thought the plan was to just have everybody install it?

This was one of the options proposed with @LuDuda and discussed at the tech writers' call, but I understood the way to go is to have a subsection under required tools. So this is the option I went for.

Reply:
However, I wasn't aware GN is supported on Windows... back to drawing board, then.